servicecontrol/configure-non-privileged-service-account.md

```diff
@@ -27,6 +27,7 @@ At a minimum, you should modify the rights to these queues:
 
 - `particular.servicecontrol`
 - `particular.servicecontrol.errors`
+- `particular.servicecontrol.staging`
```

Comment:
since this doco spans multiple versions will `particular.servicecontrol.staging always exist?